locales/en.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Services": "Services",
+    "Blog": "Blog",
+    "Case Studies": "Case Studies",
+    "About Us": "About Us",
```

Comment:
Always consider which page is hardcoded and which page is purely dynamic content